xml/apps_firefox.xml

```diff
@@ -570,7 +585,7 @@
     To edit the <literal>Bookmarks Toolbar</literal>, you can use the library
     as described in <xref linkend="sec-firefox-bookmarks-organise"/>. Its
     content is located in the folder <guimenu>Bookmarks Toolbar</guimenu>. It
-    is also possible to manage the toolbar directly. To add a folder, bookmark,
+    is also possible to manage the toolbar directly. To add a folder, bookmark
```

Comment:
At the very end of this para, there is "bookmarks toolbar' - would you replace it with the appropriate <guimenu>Bookmarks Toolbar</guimenu>? I can't edit it here.

Reply:
because of the comma: the vale stylechecker told me to remove it. Also in other places. So, who is right?